desktop/cmp/persistenceManager/PersistenceManagerModel.ts

```diff
@@ -229,6 +218,7 @@ export class PersistenceManagerModel<T extends PlainObject = PlainObject> extend
 
         this.setPendingValue(value);
         await this.onChangeAsync(value);
+        this._loadedInitially = true;
```

Comment:
I think it would be nice if we can leverage the `LoadSpec` API to check if we’ve already completed a successful load.

Reply:
That way we wouldn’t need this update this observable and could just have a getter.